### DISCUSSION TITLE
certbot inspircd ssl setup

### DISCUSSION THREAD

**Discussion by fhazal** [inspircd/inspircd#1918] [Support]
i already get my certificate with let's encrypt using certbot but my problem is i don't know what to put at inspi config to make it work with the cert,can somebody help me with the code?

inspircd version :  InspIRCd 3.10.0
certbot version : certbot 0.31.0

**Comment by sadiepowell** [comment]
There's an example script here that you can customise for use as a deploy hook with CertBot: https://docs.inspircd.org/3/modules/sslrehashsignal/#special-notes
